backend/libraries/ballerina-core/codegen/python/language constructs/union.fs

```diff
@@ -0,0 +1,57 @@
+namespace Ballerina.DSL.Codegen.Python.LanguageConstructs
+
+
+module Union =
+  open Ballerina.Core.StringBuilder
+  open Ballerina.Collections.NonEmptyList
+  open Ballerina.DSL.Codegen.Python.LanguageConstructs.Model
+
+  let private indent = (+) "    "
+
+  let private appendCaseName (allCases: string) (nextCase: string) = $"{allCases} | {nextCase}"
+
+  type PythonUnion =
+    { Name: string
```

Comment:
this one also is finicky -> could be created with an empty string

Reply:
You are about to be amazed